dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -0,0 +1,304 @@
+#!/bin/bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+usage="\e[93;1m$(basename "$0") \e[0;1m{-5} [-u <username>] [-p <passwd>] \e[90m{-t <tag>} {-r <registry>} {-n} {-s} {-h} \e[0m-- Login to dev cluster and deploy che with specific build tag
```

Comment:
You should replace `\e` with `\033` c.f. [this stackoverflow comment](https://stackoverflow.com/questions/5947742/how-to-change-the-output-color-of-echo-in-linux#comment24883926_5947802)

Reply:
done.